### ISSUE TITLE
Extra declared namespaces in SVG icons break SVG

### ISSUE THREAD

**Issue by joshma** [kisenka/svg-sprite-loader#16]
I've traced an issue down to using extra declared namespaces, where the SVG's elements aren't included in the final sprite. Here's a sample SVG file exported from Sketch, a reasonably popular design tool:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
    <!-- Generator: Sketch 3.4.3 (16618) - http://www.bohemiancoding.com/sketch -->
    <title>test</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Icons" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
        <g id="test" sketch:type="MSArtboardGroup" fill="#344043">
            <rect id="Rectangle-16" sketch:type="MSShapeGroup" x="4" y="4" width="8" height="8"></rect>
        </g>
    </g>
</svg>
```

(It's a simple square.)

It looks like svg-sprite-loader will grab the inner contents of the SVG and then insert it into a `template` that has the opening/closing SVG tags and run that through `new DOMParser().parseFromString(...)`. At this point the SVG string looks like
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol viewBox="0 0 16 16" id="dropbox"> <!-- Generator: Sketch 3.4.3 (16618) - http://www.bohemiancoding.com/sketch --> <title>test</title> <desc>Created with Sketch.</desc> <defs/> <g id="dropbox_Icons" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage"> <g id="dropbox_test" sketch:type="MSArtboardGroup" fill="#344043"> <rect id="dropbox_Rectangle-16" sketch:type="MSShapeGroup" x="4" y="4" width="8" height="8"/> </g> </g> </symbol></svg>
```

Note that there are `sketch:*` attributes like `sketch:type` present, but the `xmlns:sketch` declaration on the SVG is gone. DOMParser (on Chrome at least) will *drop* the outer `<g>` tag because of the invalid `sketch` attribute, and you end up with an empty symbol.

This isn't a blocking issue, since we can clean up our SVGs, but it would be nice if it just stripped the attributes or added the namespaces from the svg onto the template.

(Thanks for making this loader!)

**Comment by kisenka** [comment]
#11

**Comment by kisenka** [comment]
@joearasin you can use [`spriteModule`](https://github.com/kisenka/svg-sprite-loader#custom-sprite-implementation) option as quick workaround for custom sprite implementation. It can be based on [sprite.js](https://github.com/kisenka/svg-sprite-loader/blob/master/lib/web/sprite.js) and all you need to do in this case, is to add some magic in wrapSVG method.

**Comment by kisenka** [comment]
Fixed in [0.0.16](https://github.com/kisenka/svg-sprite-loader/releases/tag/v0.0.16)
